### PR TITLE
Create release-5.x branch

### PR DESCRIPTION
**Pull request by antoineco**
Create a branch for pinning the stack version to `5.x`, as requested in #467

Changes between `5.x` and `current`:

* Authentication
  * The bootstrap password is not configurable, it's always initialized to `changeme` (the logic is built into Elasticsearch, not in the Docker image).
  * All built-in users (`elastic`, `kibana`, `logstash_system`) are enabled by default, and their password is also initialized to `changeme` (also an Elasticsearch feature).
  * There is no `bin/elasticsearch-setup-passwords` utility, so passwords must be changed manually using either Kibana or the Elasticsearch API.
* Paid features (X-Pack)
  * There is no global `xpack.license.self_generated.type` setting that allows for disabling all paid features. Those have to be disabled one by one.
* Misc settings
  * `elasticsearch.hosts` -> `elasticsearch.url`
  * `xpack.monitoring.collection.enabled: true` doesn't exist
* Docs
  * A few official documentation pages are located in slightly different places.

cc @m2kar

### CODE REVIEW

**Comment by deviantony** [comment]
Seems to be some issues with that one, before I start digging into this, I would like to open a discussion regarding support for versions that are in EOL. I'll comment in the original issue.

**Comment by antoineco** [comment]
@deviantony did you clean your data volume and rebuild before starting the stack? (you never know). Travis tests run to completion.

**Comment by deviantony** [comment]
Yes, I did.

It requires some configuration changes on logstash and kibana side, but I wanted to start the EOL discussion before digging into this.

**Comment by deviantony** [comment]
> Did you clean your data volume and rebuild before starting the stack? (you never know). Travis tests run to completion.

Ahem, forgot to rebuild the images :angel: 

LGTM

**Comment by deviantony** [comment]
I'm gonna add a notice about EOL for this version

**Comment by deviantony** [comment]
cc @antoineco for validation

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 [![Join the chat at https://gitter.im/deviantony/docker-elk](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/deviantony/docker-elk?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Elastic Stack version](https://img.shields.io/badge/ELK-7.5.1-blue.svg?style=flat)](https://github.com/deviantony/docker-elk/issues/462)
-[![Build Status](https://api.travis-ci.org/deviantony/docker-elk.svg?branch=master)](https://travis-ci.org/deviantony/docker-elk)
+[![Elastic Stack version](https://img.shields.io/badge/ELK-5.6.16-blue.svg?style=flat)](https://github.com/deviantony/docker-elk/issues/467)
+[![Build Status](https://api.travis-ci.org/deviantony/docker-elk.svg?branch=release-5.x)](https://travis-ci.org/deviantony/docker-elk)
 
-Run the latest version of the [Elastic stack][elk-stack] with Docker and Docker Compose.
+Run the version 5.x of the [Elastic stack][elk-stack] with Docker and Docker Compose.
+
+**Important notice**: This release version of the Elastic stack is marked as End Of Life by Elastic and is not supported anymore, see [Elastic maintenance tables](https://www.elastic.co/support/eol) for more details.
```
[the remark this answers is not in the document]
Comment (deviantony) [reply README.md]: Even better

**Comment by deviantony** [comment]
Great job @antoineco

**Pull request merged**